Make use of the …::class feature

thiemowmde opened the pull request.

JeroenDeDauw commented:
Thiemo, what I am seeing here is the following: First it becomes clear [a certain change is controversial](https://github.com/DataValues/Common/pull/75) and then you introduce it alongside other changes in another PR without mentioning the change or the controversial bit. I am not sure how to interpret this as anything but anti-collaborative, and I feel this is disrespectful.

thiemowmde commented:
I believe you are referring to a mistake I made: I accidentally pushed two unrelated commits. I already fixed this like 30 seconds after I created the pull request. You might have seen the controversial change in the email you got, but not here when you opened this page 5 minutes after I fixed the mistake already.

Please look more carefully at what you are blaming people for.

Besides, there is nothing "controversial" in #75. It's fixing a bug.

JeroenDeDauw commented:
Great to see it was a mistake. I've seen you do this in the past without amending it, hence I did not assume it was a mistake. Glad that we both can agree doing so on purpose would be uncool.

> Besides, there is nothing "controversial" in #75. It's fixing a bug.

I'm objecting against it, so it is controversial. Or at the very least, I objected against it. That is what is important here, teamwork wise. Not the definition of a word.
